src/snowflake/cli/plugins/object/stage/manager.py

```diff
@@ -90,6 +96,13 @@ def put(
             )
         return cursor
 
+    def copy_files(self, source_path: str, destination_path: str) -> SnowflakeCursor:
```

Comment:
Should we consider adding support for 
`snow object stage copy @from_aa @to_bb`?

Reply:
Done :)
https://snowflakecomputing.atlassian.net/browse/SNOW-1213448